controllers/cart.ts

```diff
@@ -30,12 +33,15 @@ const isDiscountableItem = (item: Item) =>
 export const calculateCartController = buildEndpointRegister({
   method: RequestMethod.GET,
   path: "/cart",
+  // PRO: Usage of validations!
   validator: zod.object({
     cart: zod.object({
       reference: zod.string(),
       lineItems: zod
         .object({
           name: zod.string(),
+          // CON: I would probably also validate for negative prices. If a malicious user
```

Comment:
Agree! We probably wouldn't be sending the value of the item anyway but in this case it would definitely be benefitial